known-and-unknown.md

```diff
@@ -141,7 +141,157 @@ and a artifact (algorithm:digest).
   quickly locate the SBOM. We also learned that we are missing `hasSLSA`
   attestations for this package.
 
-2. Run the query on the Prometheus package we were working on within the workflow demo:
+2. Determine if a artifact has an SLSA attestation associated with it
```

Comment:
@sarahlinnl , can you help review this? and just put your comments inline here and @pxp928 will make the changes to it!

Reply:
Should be "Determine if an artifact..".

Reviewing the rest of this PR now.